docs/reference/aggregations/bucket/geotilegrid-aggregation.asciidoc

```diff
@@ -169,11 +169,11 @@ POST /museums/_search?size=0
 ==== Requests with additional bounding box filtering
 
 The `geotile_grid` aggregation supports an optional `bounds` parameter
-that restricts the points considered to those that fall within the
+that restricts the tiles considered to those that intersects the
```

Comment:
Tiles is probably ok here, but we also use mostly cells here so, maybe, we should replace it will cell as well for consistency.